code/modules/clothing/head/tinfoilhat.dm

```diff
@@ -6,7 +6,7 @@
 	armor_type = /datum/armor/costume_foilhat
 	equip_delay_other = 140
 	clothing_flags = ANTI_TINFOIL_MANEUVER
-	var/datum/brain_trauma/mild/phobia/conspiracies/paranoia
+	// var/datum/brain_trauma/mild/phobia/conspiracies/paranoia BUBBERSTATION CHANGE, REMOVES PARANOIA
```

Comment:
the var and all associated edits except for the removal of the trauma addition dont need to be removed from what im seeing here because it always nullchecks the trauma

Reply:
also test your prs

Reply:
no